src/studio/src/designer/frontend/dashboard/features/dashboard/index.tsx

```diff
@@ -29,12 +29,36 @@ const useStyles = makeStyles(() => ({
     fontSize: 26,
     width: 26,
   },
+  plusIcon: {
+    fontSize: 40,
+    width: 40,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  newLink: {
+    color: '#022f51',
```

Comment:
This is the same color as `theme.altinnPalette.primary.blueDarker`. Could include the theme in the makeStyles and use this defined color here 💡

Reply:
Oh, didnt know. Will fix! Thanks for the suggestion.

- [ ] use theme color